lib/libc/Kconfig

```diff
@@ -217,6 +217,19 @@ config MINIMAL_LIBC_TIME
 	  In order to make use of the non-reentrant gmtime(), it is necessary
 	  to set CONFIG_MINIMAL_LIBC_NON_REENTRANT_FUNCTIONS=y.
 
+# Note: inverted logic used to simplify conditionals in test_strerror.c
```

Comment:
Not sure why we need to change the global config scheme just for a mere test. If the rationale behind this change were to simplify the conditionals for a test, then a helper symbol should be added to the test's local Kconfig, not in the global libc Kconfig.

IMO, `MINIMAL_LIBC_STRING_ERROR_TABLE` would be better and more consistent with the rest of the configs here, especially if we make this feature disabled by default as suggested above.

Reply:
Using a local Kconfig is a smart idea. Will do that instead. Thanks!

Reply:
Done!